src/DittoMachine.sol

```diff
@@ -382,7 +402,7 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
      * @param cloneShape clone for which to compute the minimum amount.
      * @dev only use it for a minted clone.
      */
-    function _getMinAmount(CloneShape memory cloneShape) internal view returns (uint256) {
+    function _getMinAmount(CloneShape memory cloneShape, bool intraBlock) public view returns (uint256) {
```

Comment:
still should be internal?

Reply:
oh yeah, i was calling it in a test so i made it public. can change it back